docs/README.md

```diff
@@ -0,0 +1,107 @@
+# Octokit Requirements
+
+> Provide a first class experience for platform developers in their preferred programming languages
+
+Octokit strives to offer complete and officially supported SDKs, quick starts, code samples, and copy/paste-able examples for the GitHub Platform in the most popular languages.
+
+Each Octokit will work towards meeting these requirements:
+
+|                                 | JavaScript | Ruby | .NET |
+| ------------------------------- | ---------- | ---- | ---- |
+| [Complete](#complete)           | 🌔         | 🌔   | 🌔   |
+| → REST                          | ✅         | ✅   | 🌔   |
+| → GraphQL                       | 🌔         | 🌑   | 🌔   |
+| → GitHub Apps                   | 🌔         | 🌔   | ✅   |
+| → OAuth                         | 🌔         | ✅   | ✅   |
+| → Webhooks                      | ✅         | 🌑   | 🌔   |
+| [Decomposable](#decomposable)   | ✅         | 🌑   | 🌑   |
+| [Generated](#generated)         | ✅         | 🌔   | 🌔   |
+| [Native](#native)               | ✅         | ✅   | ✅   |
+| [Documented](#documented)       | 🌔         | ✅   | 🌔   |
+| [Tested](#tested)               | ✅         | 🌔   | 🌔   |
+| [Friendly](#friendly)           | 🌑         | 🌑   | 🌑   |
+| [Prescriptive](#prescriptive)   | 🌑         | 🌑   | 🌑   |
+| [Supported](#supported)         | ✅         | 🌑   | 🌑   |
```

Comment:
> New features are supported in a stable release of the SDK at the time of the feature being released.
> Developers will receive code-level technical support on client libraries from GitHub.

@gr2m This is close, but I think it's still 🌔 since the releases still aren't in sync and we don't officially offer code-level support.